dll/win32/shell32/shelldesktop/CDesktopBrowser.cpp

```diff
@@ -458,6 +460,22 @@ LRESULT CDesktopBrowser::OnGetChangeNotifyServer(UINT uMsg, WPARAM wParam, LPARA
     return (LRESULT)m_hwndChangeNotifyServer;
 }
 
+extern VOID WINAPI ShowFolderOptionsDialog(UINT Page, BOOL Async);
```

Comment:
```suggestion
extern VOID ShowFolderOptionsDialog(UINT Page, BOOL Async);
```

Reply:
This file is compiled by both shell32 and rshell IIRC so that is why I chose to add a calling convention.

Reply:
Interesting thought you spent @whindsaks .
Have you tested, Does it actually break/hurt rshell if you strip the calling convention here? If yes in which exact case/mixture of builds?

Reply:
I don't remember specifically.

Either way, whatever tiny overhead stdcall might have here is not worth thinking about.